contentcuration/contentcuration/assets/channelList/vuex/channelList/getters.js

```diff
@@ -0,0 +1,37 @@
+import omit from 'lodash/omit';
+import isEqual from 'lodash/isEqual';
+import { isDirty, isDummyId } from '../../utils';
+
+export function channels(state) {
+  return Object.values(state.channelsMap).filter(channel => !isDummyId(channel.id));
```

Comment:
Our base vuex state is an id: object map (but a literal JS `object`).

To return all channels we just get the values, but we filter out any dummy channels that have not been saved yet.

Could also add a default sort order in here.